### PR TITLE
Setup base repository structure

### PR DESCRIPTION
**Pull request by webinterpret-datascience**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.6
```
Comment (patrykjarzyna) [inline setup.cfg]: Is it the right format?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+project_urls =
+    Bug Tracker = https://github.com/webinterpret-datascience/kedro-templar/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
```
Comment (patrykjarzyna) [inline setup.cfg]: Should we keep it as multiline?

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel"
```
Comment (patrykjarzyna) [inline pyproject.toml]: Should we also specify the version here?

File: setup.cfg (hunk 1 above)
Comment (webinterpret-datascience) [reply setup.cfg]: yes, but I will change it anyway to the one from https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/

**Pull request merged**